components/LanguageSwitcher/LanguageSwitcherSubMenu.vue

```diff
@@ -60,7 +42,15 @@ export default {
     },
   },
   methods: {
-    useGetAbsoluteUrlIfSwitchWebsite,
+    getIndexUrl(menuItem) {
+      const locale = language.locales.find(
+        (locale) => locale.code === menuItem.lang
+      )
+      return getAbsoluteUrlIfSwitchWebsite({
+        relativeTarget: `/${locale.code}`,
```

Comment:
```suggestion
        relativeTarget: locale.code === 'fr-fr ? '/' :`/${locale.code}`,
```

Reply:
Si je ne me trompe pas le commit bd29d9c9992112ba55afe691647b8a98391cd4fd introduit un changement de comportement. Ne faudrait-il pas mettre à jour cette PR pour indiquer qu'il ne s'agit plus uniquement d'un refacto mais plutôt d'un refacto+« feat: redirect to root for fr-fr locale » ?

Reply:
C'est déjà le cas en prod, `fr-fr` = pix.fr et donc il n'y a pas de /fr-fr

Reply:
En fait ce commit bd29d9c touche du nouveau code apporté par l'utilisation de `getAbsoluteUrlIfSwitchWebsite`. Ce n'est donc pas un changement de comportement par rapport à la prod actuellement. J'avais mal compris cela en lisant les commits séparément, désolé. Et le fait que ce commit ait pour titre `feat` a fini de malmener ma compréhension. Il aurait presque fallu que le titre soit `fix-feat: redirect to root for fr-fr locale` ;-)

Reply:
Je peux le réinclure dans les commits aussi, car c'est vrai que c'est un changement de comportement dans le refacto

Reply:
Si cela ne te gêne pas trop ce serait bien :+1: